core/src/main/java/org/fao/geonet/kernel/security/ldap/README.md

```diff
@@ -1,7 +1,7 @@
 Using LDAPUserDetailsContextMapperWithProfileSearchEnhanced
 ===========================================================
 
-I've modified `LDAPUserDetailsContextMapperWithProfileSearch` so it works in more situations, is much simpler, uses strategy objects, and has a good set of test cases.
+`LDAPUserDetailsContextMapperWithProfileSearch` was modified so it works in more situations, is much simpler, uses strategy objects, and has a good set of test cases.
 
 The original `LDAPUserDetailsContextMapperWithProfileSearch` was extremely difficult to understand.
```

Comment:
Not sure this adds value :P Even if true ....

Reply:
> Not sure this adds value

It doesn't. You could even add a blame link to the file to be even more kindfull with past contributors who tries to make things better :/